README.md

```diff
@@ -49,6 +49,12 @@ For a graphical user interface in a desktop application, see the [napari plugin]
 Acknowledgements
 ----------------
 
+ITKElastix was developed in part by with support from:
```

Comment:
Just to be sure, is `"in part by with support"` correct English? Otherwise I guess the word "with" may be removed.

Reply:
Yes, we need some swapping!